src/components/OpeningDate.js

```diff
@@ -46,9 +46,59 @@ const OpeningDate = (props) => {
       {props.cumulativeFullAnswers.establishment_opening_status ===
       "Establishment due to trade" ? (
         <PostForm action={props.formAction} csrfToken={props.csrfToken}>
```

Comment:
Once this is combined with the headings change the fieldset legend should be the h1 headnig.